yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/util/AnchorCertificateUtils.java

```diff
@@ -0,0 +1,138 @@
+package com.yoti.api.client.spi.remote.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.bouncycastle.asn1.ASN1InputStream;
+import org.bouncycastle.asn1.ASN1OctetString;
+import org.bouncycastle.asn1.ASN1Primitive;
+import org.bouncycastle.asn1.ASN1TaggedObject;
+import org.bouncycastle.asn1.DEROctetString;
+import org.bouncycastle.asn1.DLSequence;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.protobuf.ByteString;
+import com.yoti.api.client.spi.remote.proto.AttrProto.Anchor;
+
+public class AnchorCertificateUtils {
```

Comment:
Do you mind to spit this class in two classes in order to avoid `....Utils` classes? 
The risk is that we don't define the correct responsibilities of the class and we ended up to put everything related to an AnchorCertificate in there, with the result of a difficult understanding of the code that uses these methods. It's just my opinion ofc. 

So one class will be responsible for the mapping between Achor and AncorVerifierSourceData and the other one for `combineBinaryCertChain`. Thanks

Reply:
Got rid of `combineBinaryCertChain`